net/cert/pki/test_helpers.cc

```diff
@@ -414,7 +423,11 @@ bool ReadVerifyCertChainTestFromFile(const std::string& file_path_ascii,
 std::string ReadTestFileToString(const std::string& file_path_ascii) {
   // Compute the full path, relative to the src/ directory.
   base::FilePath src_root;
+#if defined(STARBOARD)
+  base::PathService::Get(base::DIR_TEST_DATA, &src_root);
```

Comment:
unsure about DIR_TEST_DATA vs DIR_SOURCE_ROOT, can we use upstream somehow for tests?